src/mainwindow.cpp

```diff
@@ -479,6 +487,12 @@ void PointViewerMainWindow::addFiles()
     m_currFileDir.makeAbsolute();
 }
 
+void PointViewerMainWindow::finalSetup()
+{
+    openShaderFile("shaders:las_points.glsl");
```

Comment:
Maybe let's put this in as the default inside `View3D`, rather than bothering with the extra complexity the `initializedGL` -> `finalSetup` signal slot connection.